src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java

```diff
@@ -50,32 +57,32 @@
 
 @SpringBootApplication
 // Warning! This implies @ComponentScan - and we really must have that in place, e.g. due to JMS :(
-
+//@EnableAutoConfiguration
```

Comment:
Remove it, if you don't need it

Reply:
Glad you pointed this out. I did a spike to see if I could remove line 58 (and the @ComponentScan). It looked like it mostly worked. I've removed the commented out code, but will revisit this some time.